javascript.yml

```diff
@@ -0,0 +1,39 @@
+# Since the jobs have their stages defined here, makes sense for this code to own what stages can exist.
+stages:
+  - lint
+  - build
+  - unitTest
+  - deploy
+
+
+.js_lint:
+  stage: lint
+  variables:
+    eslint_version: "6"
+  script:
+    - docker run --rm -v $(pwd):/data cytopia/eslint:$eslint_version .
+
+.js_build:
+  stage: build
+  variables:
+    node_version: "13.12.0"
```

Comment:
I wonder if this could be tied to the version of node that the consumer of the pipeline is using?